src/main.py

```diff
@@ -18,11 +18,11 @@
 import gi
 import sys
 
+gi.require_version('Gtk', '3.0')
```

Comment:
could you add` # noqa: E402` at the end of the line to remove the flake8 error (like in the branch [version3](https://github.com/lachhebo/GabTag/blob/88374e1cc3760d1c64893b6cb95097eecd5ec483/src/main.py#L25))